magento/magento2
# Grammar fix for #9533

### Description

Quick fix for verbiage in the authentication popup, changed "checkout out" to "checkout" in two places.

### Fixed Issues (if relevant)

1. magento/magento2#9533: Grammar error on authentication popup

### Manual testing scenarios

1. On the frontend with guest checkout disabled, add an item to your cart and proceed to checkout.
2. The authentication popup that appears has two sections. Validate that the one on the left has a header that says "Checkout as a new customer".
3. Validate that the one on the right has a header that says "Checkout using your account".
